FerramAerospaceResearch/FARGUI/FAREditorGUI/Simulation/InstantConditionSim.cs

```diff
@@ -141,15 +141,15 @@ public void GetClCdCmSteady(InstantConditionSimInput input, out InstantCondition
             AngVel += (sinPhi * alphaDot - cosAlpha * cosPhi * betaDot) * up;
 
             Vector3d velocity = forward * cosAlpha * cosBeta;
-            velocity += right * (sinPhi * cosAlpha * cosBeta + cosPhi * sinBeta);
-            velocity += -up * cosPhi * (sinAlpha * cosBeta + sinBeta);
-
-            velocity.Normalize();
+            velocity += right * (sinPhi * sinAlpha * cosBeta + cosPhi * sinBeta); // Rodhern: Changed velocity 'right' and 'up' components
+            velocity += -up * (cosPhi * sinAlpha * cosBeta - sinPhi * sinBeta);   //  to align vector to naive forward direction.
 
             //this is negative wrt the ground
             Vector3d liftVector = -forward * sinAlpha + right * sinPhi * cosAlpha - up * cosPhi * cosAlpha;
 
-            Vector3d sideways = Vector3.Cross(velocity, liftVector).normalized;
+            Vector3d sideways = -forward * cosAlpha * sinBeta;
+            sideways += right * (cosPhi * cosBeta - sinPhi * sinAlpha * sinBeta); // Rodhern: The same as Vector3.Cross(velocity, liftVector)
+            sideways += up * (cosPhi * sinAlpha * sinBeta + sinPhi * cosBeta);    //  but possibly with better accuracy.
```

Comment:
It's better to leave it as it was because Vector3.cross is more concise and better for readability unless the accuracy is much worse

Reply:
Sure I can change that. Do you feel it is better for readability even when the remark is in the code? I feel there is sort of a nice analogous feel between 'velocity', 'liftVector' and 'sideways' this way. It is up to you though.

Reply:
We're computing principal axes so cross is easier and shows that explicitly, no one will ever bother computing the last component from scratch (at least on paper) when it is also perpendicular to the other two